Fix typo ... variable is named 'SearchDate' not 'date'

patch by Sheldon Hearn
http://bugs.gentoo.org/96362

--- scripts/logfiles/emerge/applydate
+++ scripts/logfiles/emerge/applydate
@@ -31,7 +31,7 @@
 
 if ($Debug > 5) {
     print STDERR "DEBUG: Inside applydate (emerge)...\n";
-    print STDERR "DEBUG: Looking For: $date\n";
+    print STDERR "DEBUG: Looking For: $SearchDate\n";
 }
 
 # Examine each line of the file, writing out only the lines that are within
